### PR TITLE
[UI-42] form wrapper

### PR DESCRIPTION
**Pull request by maciek15**
#### Task

- **Issue or task referred:** UI-42
- **Feature or Bugfix:** Feature

It is a simple form component with footer created according to provided mocks.

#### Checklist

- [x] Required unit tests prepared
- [x] Documentation prepared
- [x] External licenses validated (ideally MIT)

### CODE REVIEW

**Comment by rangoo94** [comment]
Remember about marking commits with task ID.

File: packages/form/styles/main.sass [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import './geometry'
```
Comment (rangoo94) [inline packages/form/styles/main.sass]: Remove `storybook.sass`

File: packages/form/stories.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    </ControlGroup>
+  )
+}
+
+// Stories
+
+addStory('default', readme, () => (
+  <Form onSubmit={e => e.preventDefault()}>
+    <h2>This is a default form wrapper</h2>
+    <TextInput placeholder='Name' left={<Icon name='person' />} /><br />
```
Comment (rangoo94) [inline packages/form/stories.js]: It should use `FormField` when will be ready.

File: packages/form/src/Form.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  const contentCls = buildClassName([moduleName, 'content'])
+  const footerCls = buildClassName([moduleName, 'footer'])
+
+  return (
+    <form className={formCls} {...passedProps}>
+      <div className={contentCls}>
+        {children}
+      </div>
+      {
+        footerComponent &&
```
Comment (michalochman) [inline packages/form/src/Form.js]: Was there a requirement to have footer treated in a special way? Why can't we just add it to children?

File: packages/form/styles/config.sass [hunk 4]
```diff
@@ -0,0 +1,3 @@
+@import '~@talixo/shared/styles/config'
+
+$color-border: #662FB3
```
Comment (stuniel) [inline packages/form/styles/config.sass]: ( ͡° ͜ʖ ͡°)

File: packages/form/styles/theming.sass [hunk 5]
```diff
@@ -0,0 +1,7 @@
+@import './config'
+
+@include module('form')
+  display: block
```
Comment (PiotrGrzechnik) [inline packages/form/styles/theming.sass]: It's not necessary here, already in geometry.sass

**Pull request merged**